understanding/22/accessible-authentication-minimum.html

```diff
@@ -105,10 +105,10 @@ <h3>Copy and paste</h3>
         </section>
 
         <section id="two-factor-authentication">
-            <h3>Two-factor authentication systems (Temporary code / One-Time Password)</h3>
-            <p>Beyond usernames and passwords, some sites may use two-factor authentication, asking the user to enter a temporary code or One-Time Password (OTP). A service that requires <em>manual</em> transcription of a 2FA code is not compliant. As with usernames and passwords, it must be possible for a user to at least paste the code or <abbr title="One-Time Password">OTP</abbr> (such as a standalone third-party password manager, text message application, or software-based security key), or to allow user agents to fill in the fields automatically.</p>
-            <p>There are scenarios where a temporary code or One-Time Password (OTP) must be received or generated on a secondary device. For example, authenticating in a web browser on a laptop requires an OTP that is sent as an SMS text message to a mobile phone. However, in most cases, it is possible for the temporary code or OTP to then be sent directly to the primary device, where it can then be copied and pasted – for example, by copying the OTP on the secondary device and emailing it to the primary device, or through the use of a shared cross-device clipboard where copying content on the secondary device makes it available to paste on the primary device. Evaluating whether or not the temporary code/OTP can be seamlessly transferred from the secondary device to the primary device is <em>outside of the scope</em> for this Success Criterion. For the purpose of evaluating web content that relies on authentication using these types of secondary device systems, we assume that provisions are in place that make the code/password available in the user's clipboard – evaluating this criterion therefore only requires verification that the web content does allow pasting the clipboard content in the related authentication challenge field.</p>
-            <p>Note that two-factor systems that do not rely on codes or passwords – including hardware authentication devices (such as YubiKey), secondary applications (either on the same primary device, or on a secondary device) that expect the user to confirm that it is indeed them trying to log in, and authentication methods provided by the user's operating system (such as Windows Hello, or Touch ID/Face ID on macOS and iOS) - are <em>not</em><a> a cognitive function test</a>.</p>
+            <h3>Two-factor authentication systems (verifcation codes)</h3>
```

Comment:
noting though that TOTP (time-based one-time password) / OTP (one-time password) are actual acronyms/terms that are used in practice when talking about these...codes

Reply:
ah, I see you're then mentioning/namechecking it in the first sentence. I'd still love to see the acronym itself also presented/mentioned, but not too precious about it. Also might be nice to, parenthetically, mention 2FA / MFA? or is that too technical?

Reply:
Normally one only uses an acronym if the intent is to reuse later, and it's not reused here. Since I'm using the phrases in parentheses already, it's a bit disruptive to try to work in the acronym too.